### PR TITLE
Refactor tokens actions and reducer into redux toolkit version

### PR DESCRIPTION
**Pull request by marcinbodnar**

### CODE REVIEW

File: packages/frontend/src/utils/tokens.js
Comment (MaximusHaximus) [inline packages/frontend/src/utils/tokens.js:5]: This file should be deleted rather than modified -- it has been _replaced_ with `FungibleTokens` service https://github.com/near/near-wallet/blob/cffe2d4829ffb9c42efe4853ae2b5a1609b64a75/packages/frontend/src/services/FungibleTokens.js#L44, whose associated functions already have the correct signature and return values.

File: packages/frontend/src/actions/account.js
Comment (MaximusHaximus) [inline packages/frontend/src/actions/account.js:463]: Pretty sure these lines are gone as soon as we land prior PR, so this shouldn't be part of this diff

File: packages/frontend/src/redux/slices/tokens/index.js
Comment (MaximusHaximus) [inline packages/frontend/src/redux/slices/tokens/index.js:98]: Should we be re-setting `error` value in either of these?  I would assume we'd at least want to in `.fulfilled`?

**Review by MaximusHaximus — CHANGES_REQUESTED** [review]
As with other PRs, this has a merge commit that's adding a few thousand lines of diff from `master` into this feature branch's commits.  Other than that, just a couple things in-line! :)

File: packages/frontend/src/utils/tokens.js
Comment (marcinbodnar) [reply packages/frontend/src/utils/tokens.js]: Done.

File: packages/frontend/src/actions/account.js
Comment (marcinbodnar) [reply packages/frontend/src/actions/account.js:462]: Done.

File: packages/frontend/src/redux/slices/tokens/index.js
Comment (marcinbodnar) [reply packages/frontend/src/redux/slices/tokens/index.js]: Done.

File: packages/frontend/src/actions/account.js
Comment (MaximusHaximus) [reply packages/frontend/src/actions/account.js:462]: Looking at the current `master`, it appears that this code is actually still in the codebase now; I think this is a bug since the _handlers_ for these actions were deleted in #2002 (now these dispatch calls are no-ops) -- please create a separate fix for this problem, we doesn't need to block this PR and wouldn't want to roll a bug-fix into this anyhow :)

**Review by MaximusHaximus — APPROVED** [review]
Looks like there's a conflict to resolve, otherwise this is ready to land! <3 :)

I added one comment in-line about a bug introduced in #2002 because the `clearState()` dispatches in `finishAccountSetup()` were not removed / replaced with the new `clearAccountState` dispatch logic, but the listeners to the previous `CLEAR_STATE` actions were removed -- that should be fixed ASAP in it's own PR since that is now a live bugs since the handlers were removed in #2002 :)

**Pull request merged**
